### PR TITLE
🐛 FIX: Typos & Grammar

### PR DESCRIPTION
**Pull request by ahmadawais**
Fixes a couple of typos and grammatical errors like:

```diff
-	editting
+	editing
-	Nesting allow
+	Nesting allows
-	some people find the nesting difficult to visually distinguish
+	some people find the nesting challenging to distinguish visually
```
Looking forward, peace! ✌️

### CODE REVIEW

**Comment by ahmadawais** [comment]
@frivoal First time here, kindly, let me know if there's anything I can do to pass the check, can't seem to have the ability to mark this PR as non-substantive.

**Comment by ahmadawais** [comment]
Marked as non substantive for IPR from ash-nazg.

File: css-nesting-1/Overview.bs [hunk 1]
```diff
@@ -270,14 +270,14 @@ The Nesting At-Rule: ''@nest'' {#at-nest}
 	Some valid nesting selectors,
 	like ''.foo &'',
 	are disallowed,
-	and editting the selector in certain ways can make the rule invalid unexpectedly.
+	and editing the selector in certain ways can make the rule invalid unexpectedly.
 	As well,
-	some people find the nesting difficult to visually distinguish
+	some people find the nesting challenging to distinguish visually
```
Comment (tabatkins) [inline css-nesting-1/Overview.bs]: This line wasn't wrong, but the "correction" isn't wrong either, so I'll go ahead and accept anyway. ^_^

Comment (ahmadawais) [reply css-nesting-1/Overview.bs]: Thank you! 👌

**Comment by ahmadawais** [comment]
Thank you for merging this PR! 👌

**Pull request merged**